src/routes/RemediationDetailsComponents/ActionsContent/ActionsContent.js

```diff
@@ -1,23 +1,25 @@
-import React, { useCallback, useMemo, useState } from 'react';
-import RemediationsTable from '../../../components/RemediationsTable/RemediationsTable';
+/* eslint-disable react/prop-types */
```

Comment:
🤨 (there is no friendly version of this emoji, it should be a friendly eyebrow raise.)

Reply:
Component: ({ systems, description }) =>

the props on this line scream, and the proper fix is to move the inline component out of things and name it, but that cause another slew of issues that causes everything to get super ugly

Reply:
ehh i changed it anyways